cider/tests/utils/CiderTestBase.cpp

```diff
@@ -37,6 +37,9 @@ void CiderTestBase::assertQuery(const std::string& sql,
   auto cider_input = json_file.size() ? json_file : sql;
   auto cider_res_batch = std::make_shared<CiderBatch>(
       ciderQueryRunner_.runQueryOneBatch(cider_input, input_[0]));
+
+  std::cout << cider_res_batch->toValueString() << std::endl;
```

Comment:
please remove

Reply:
There is an UT failure I can't reproduce locally so add these lines for some output. Will remove once resolved.